test-project/Assets/Config/GdkToolsConfiguration.json

```diff
@@ -1,7 +1,11 @@
 {
-    "SchemaStdLibDir": "build/dependencies/schema/standard_library",
+    "SchemaStdLibDir": "../../build/dependencies/schema/standard_library",
```

Comment:
This puts it outside of your project directory I think: 

```
gdk-for-unity/
	test-project/
		build/ <-- Previous location
build/ <-- New location
```

Reply:
you are correct

made a mistake when copying across from playground